### DISCUSSION TITLE
RuCore / Расширения / Core_Ex_FIAS_House

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#239] [General]
Указание дома перенесено в комплексное расширение - [238](https://github.com/fhir-ru/core/discussions/238), на основании решений описанных в [242](https://github.com/fhir-ru/core/discussions/242#discussioncomment-7457586).

**Comment by Vrodvi** [comment]
1. В идентификаторе не обозначено требование на указание пространства идентификаторов ФИАС 

Следует указать то пространство, которое указано в специально для этого созданном ресурсе - https://github.com/fhir-ru/core/discussions/242.

**Comment by Vrodvi** [comment]
2. Кардинальность значения позволяет не указывать идентификатор

Кардинальность позволяет не указать идентификатор, но при этом нет никакого аналога nullFlavor или ещё каких-либо способов обозначить причину отсутствия идентификатора.
В СЭМДах это решалось требованиями вида ДОЛЖЕН с возможностью обозначить nullFlavor. Какую замену сообщество предлагает вместо этого?

**Comment by Vrodvi** [comment]
3. Структура идентификации по ФИАС отличается от СЭМДов

В СЭМДах для идентификации ФИАС используется комплексное расширение с двумя атрибутами - идентификатор адресного объекта и идентификатор дома. Почему структура сделана отличной от СЭМДов? Почему наименование атрибутов сделано отлично от СЭМДов (там это HOUSEGUID, тут core-ex-address-fias-house)?

Вот тут в деталях уже это было разобрано и была предложена аналогичная СЭМДам комплексная структура - https://github.com/fhir-ru/core/discussions/83.

Ожидается, что структура идентификации по ФИАС будет приведена к структуре СЭМДов или будет явно обозначено обоснование расхождения и дана схема маппирования предложенной структуры на структуру СЭМДов.

**Comment by Vrodvi** [comment]
4. В представленных фрагментах кода (FSH) сбивается форматирование 

К примеру символ звёздочки превратился в обозначение элемента списка.
Для предотвращения искажений форматирования кода просьба все кодовые фрагменты оборачивать в специальную разметку - тройной символ апострофа.
Пример:
если указать:
\```
\* фрагмент кода 
\```
, то будет корректное отображение:
```
* фрагмент кода 
```
, а не 
* фрагмент кода

**Comment by eugenekogan-spb** [comment]
Указание дома перенесено в комплексное расширение - [238](https://github.com/fhir-ru/core/discussions/238), на основании решений описанных в [242](https://github.com/fhir-ru/core/discussions/242#discussioncomment-7457586).
